content/en/blog/_posts/2017-09-00-Kubernetes-Statefulsets-Daemonsets.md

```diff
@@ -4,6 +4,9 @@ title: " Kubernetes StatefulSets & DaemonSets Updates "
 date: 2017-09-27
 slug: kubernetes-statefulsets-daemonsets
 url: /blog/2017/09/Kubernetes-Statefulsets-Daemonsets
+author: >
+   Janet Kuo (Google),
+   Kenneth Owens (Kenneth Owens)
 ---
 Editor's note: today's post is by Janet Kuo and Kenneth Owens, Software Engineers at Google.
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.